SPECS/python-requests/python-requests.spec

```diff
@@ -35,6 +38,7 @@ BuildRequires:  python-attrs
 BuildRequires:  python-certifi
 BuildRequires:  python-chardet
 BuildRequires:  python-idna
+BuildRequires:  python-pip
 BuildRequires:  python-urllib3
 %endif
 %if %{with_check}
```

Comment:
Do you know if there's a reason why we have two `if %{wich_check}` sections here? Could we merge them into one?

Reply:
The first is for python2 requirements, and second is for python3

Reply:
Will keep this as-is for now